README.md

```diff
@@ -36,10 +36,7 @@ To find the git SHA of any given release, check the latest commit in the version
 |--------------|--------------------|
 | Latest Stable| `https://downloads.dcos.io/dcos/stable/dcos_generate_config.sh` |
 | Latest Master Linux| `https://downloads.dcos.io/dcos/testing/master/dcos_generate_config.sh` |
-| Latest Master Windows| `https://downloads.dcos.io/dcos/testing/master/windows/dcos_generate_config_win.sh` |
 | Latest Build of Specific PR Linux| `https://downloads.dcos.io/dcos/testing/pull/<github-pr-number>/dcos_generate_config.sh` |
```

Comment:
Remove `Linux` from this table as well. So just `Latest Master` and `Latest Build of Specific PR`

Reply:
done